kahuna/public/js/preview/image.html

```diff
@@ -75,18 +75,20 @@
                 {{::ctrl.image.data.uploadTime | date:'HH:mm'}}
             </span>
 
-            <span class="bottom-bar__meta-item titip-bottom preview__in-collection"
-                ng:if="ctrl.showCollections && ctrl.image.data.collections.length > 0">
-                <gr-icon gr-small>folder</gr-icon>
-                <span class="titip-content preview__in-collection__list">
-                    <li ng-repeat="collection in ctrl.image.data.collections">
-                        <a ui:sref="search.results({query: '~'+collection.data.pathId})">
-                            {{collection.data.path.join(' ▸ ')}}
-                        </a>
-                        <br>
-                    </li>
-                </span>
-            </span>
+            <ul class="bottom-bar__meta-item titip-bottom preview__collections">
+                <li class="preview__collections__collection"
+                    ng:repeat="collection in ctrl.image.data.collections"
+                    style="background-color: {{collection.data.cssColour}}">
+                    <a ui:sref="search.results({query: '~'+collection.pathId})"
```

Comment:
Could we run this through the [`querySyntax`](https://github.com/guardian/grid/blob/master/kahuna/public/js/search-query/query-syntax.js#L26-L28) helper?